contents/handbook/engineering/posthog-com/changelog.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Changelog entries
+---
+
+The [changelog](/changelog) is run by the [Word & Pictures](/teams/words-pictures) Team.
+
+Changelog entries can be created in two ways:
+
+1. Change the status of an existing [roadmap](/roadmap) or [WIP](/wip) entry to "Complete"
+1. Create a new changelog entry on the [changelog](/changelog) page.
+
+Both options are available when you're signed into posthog.com as a moderator.
+
+## Fill out the proper paperwork
+
+Make sure all fields are filled out correctly before creating an entry.
+
+### Date
+
+Set the date to the feature's release date. If updating from a previous roadmap or WIP entry, change the date to the actual release date (rather the date where the team started working on the feature).
+
+### Team and author
+
+Select the team that is responsible for the change, and choose the author of the feature. If there's no individual lead on the feature or update, set the author to the team lead.
+
+### Categorization
+
+Select the product or product area that the change is relevant to, and set the type of update. These can be used on the front end to filter down to a subset of changes.
+
+### Title and description
+
+Be succinct with the title. Check the format of existing entries for inspiration.
+
+### Options
+
+The **Show on homepage** option aggregates the entry to the _"We ship weirdly fast"_ calendar on the hompepage. Only select this option if the milestone is impressive enough to be remembered years from now. The point of the calendar is to show the frequency of shipping big features, not to highlight every single update.
+
+### Social sharing
+
+The [Website & Vibes](/teams/website-vibes) team has created an image generator that takes the information from the changelog entry and creates a square image for sharing on social media. [Read the blog post](/blog/changelog-image-generator) to learn how it works.
+
+The customization options are designed to allow you to format the copy and image so it looks as good as it can. If you need suggestions are aren't sure if your changelog image meets our quality standards, don't hesitate to post in <PrivateLink url="https://posthog.slack.com/archives/C01V9AT7DK4">#team-website-and-vibes</PrivateLink> for a second opinion.
```

Comment:
```suggestion
The customization options are designed to allow you to format the copy and image so it looks as good as it can. If you need suggestions or aren't sure if your changelog image meets our quality standards, don't hesitate to post in <PrivateLink url="https://posthog.slack.com/archives/C01V9AT7DK4">#team-website-and-vibes</PrivateLink> for a second opinion.
```